package.json

```diff
@@ -137,7 +137,7 @@
 		"typedoc-plugin-markdown": "^3.17.0",
 		"typedoc-plugin-mdn-links": "^2.0.0",
 		"typedoc-plugin-merge-modules": "^5.1.0",
-		"typescript": "^4.7.4",
```

Comment:
will any of breaking changes in TS will affect users: 
https://github.com/microsoft/TypeScript/wiki/API-Breaking-Changes

https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#breaking-changes-and-deprecations

Reply:
As we are updating at devDependencies, there should be no issues with the braking changes except with our code that I needed to update at few places for the detected issues.

But for the users of our lib there should be, theoretically no issues at all. However, there could be some very edge case that I am not aware of. But according to my search and up to my knowledge, there should be no issues for the users.

Reply:
According to my reading, the generated JavaScript is only affected by the targeted ECMA Script. And for the same ECMA Script, the transpiled code is compatible. I checked few of the generated JavaScript source code and things seems almost identical and 100% compatible.